.nais/kafka/dialogmelding.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  annotations:
+    dcat.data.nav.no/title: "Dialogmeldinger til fastleger og andre behandlere"
+    dcat.data.nav.no/description: >-
+      Topic inneholder dialogmeldinger til fastleger og andre behandlere.
+  name: isdialogmelding-dialogmelding
```

Comment:
Føler det blir feil å si at dette topic'et innheolder dialogmelding, er mer en bestilling av/sending av inn til en dialgomelding. Synes vi burde ha mer beskrivende navn/description.

Reply:
Tanken er at topic skal inneholde "hendelser" som bestiller sending av dialogmelding til behandler? Så kanskje isdialogmelding-behandler-dialogmelding-bestilling ? Eller bare behandler-dialogmelding-bestilling hvis man ikke trenger app-prefiks.

Reply:
Det kommer litt an på hvor generisk dette skal være. Skal topic'et bare brukes for å sende meldinger til behandler som er relatert til dialogmøter, eller skal det brukes mer generelt? 

Noen muligheter:
isdialogmelding-behandler-dialogmote-notifikasjon
isdialogmelding-behandler-melding
isdialogmelding-behandler-varsel

Evnt. så blir det som Anders sier at vi ikke sier noe i topic-navnet om innholdet, men i stedet refererer til at dette er en bestilling av en dialogmelding.

Reply:
Jeg tenker at vi prøver å lage en topic som kan brukes til å bestille sending av alle typer dialogmeldinger(så langt det er mulig). Da blir det lettere å vedlikeholde og ta i bruk av produsenter til topic.

Reply:
OK, nytt forsøk pushet nå.